.travis.yml

```diff
@@ -2,9 +2,10 @@ language: python
 sudo: false
 python:
     - 2.7
+    - 3.3
```

Comment:
My opinion is, support all current active versions, and additional all inherit versions used in major Enterprise Linux (RHEL, CentOS, Debian, Ubuntu LTS).
But we should add a Comment when we will drop support, which could be Sept 2017.